src/mockSdk.js

```diff
@@ -14,90 +14,97 @@ export const mockSDKPerson = {
   orgId: 'orgID',
 };
 
-export const mockSDKMeeting = {
-  id: 'meetingID',
-  sipuri: 'my meeting',
-  members: {
-    membersCollection: {
-      members: {
-        person1: {
-          name: 'Barbara German',
-          isInMeeting: true,
-          isUser: true,
-          isAudioMuted: false,
-          isContentSharing: false,
-          isHost: false,
-          isGuest: false,
-          isSelf: true,
-          participant: {
-            person: {
-              ...mockSDKPerson,
-              id: 'id',
+/**
+ * Creates a mock instance of the Webex SDK Meeting used in unit testing
+ *
+ * @returns {object} mockSDKMeeting Instance
+ */
+export function createMockSDKMeeting() {
```

Comment:
in order to prevent all the indentation changes, can you please switch to the arrow function syntax?

```js
export const createMockSDKMeeting = () => ({
  id: 'meetingID',
  ...,
});